#!/usr/bin/env python
PACKAGE = "feedback_cvsa"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("threshold_0", double_t, 0, "Threshold 0", 0.7, 0.5, 1.0)
gen.add("threshold_1", double_t, 0, "Threshold 1", 0.7, 0.5, 1.0)
gen.add("threshold_2", double_t, 0, "Threshold 2", 0.7, 0.5, 1.0)
gen.add("threshold_3", double_t, 0, "Threshold 3", 0.7, 0.5, 1.0)

exit(gen.generate(PACKAGE, "feedback_cvsa", "CVSA"))
